.github/pull_request_template.md

```diff
@@ -8,6 +8,6 @@
 
 ## Checks
 
-- [ ] updated changelog
-- [ ] updated documentation
-- [ ] updated tests 
+-   [ ] updated changelog
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/Mjolnir-Forge/mjolnir-core/pullRequest?prid=9403582)
```suggestion
- [ ] updated changelog
```